lib/postgrex/replication.ex

```diff
@@ -101,10 +113,13 @@ defmodule Postgrex.Replication do
           database: "demo_dev",
           username: "postgres",
         )
-      Postgrex.Replication.create_slot(pid, "postgrex", :pgoutput)
-      Postgrex.Replication.start_replication(pid, "postgrex",
+
+      Postgrex.Replication.start_replication(
+        pid,
+        "postgrex",
```

Comment:
```suggestion
        slot_name: "postgrex",
```
I forgot to update this in the last PR.